.github/workflows/macos-build.yaml

```diff
@@ -0,0 +1,54 @@
+name: Build on macOS
+on: push
+
+jobs:
+  build:
+    runs-on: macos-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+      #- name: csrutil disable
+      #  run: |
+      #    sudo csrutil disable
+      #- name: csrutil enable
+      #  run: |
+      #    sudo csrutil enable --without kext
+      #- name: spctl kext-consent disable
+      #  run: |
+      #    sudo spctl kext-consent disable
+      - name: install deps
+        run: |
+          brew install automake libtool gawk coreutils
+      - name: install deps
+        run: |
+          #brew install openssl@3
+      - name: autogen
+        run: |
+          ./autogen.sh
+      - name: configure
+        run: |
+          #https://stackoverflow.com/a/62591864
+          ./configure CPPFLAGS="-I/usr/local/opt/gettext/include -I/usr/local/opt/openssl/include" LDFLAGS="-L/usr/local/opt/gettext/lib/ -L/usr/local/opt/openssl/lib"
+      - name: build
+        run: |
+          make -j 2
```

Comment:
[GitHub-hosted runners](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources) have 3 vCPUs, but you are limiting `make` to only 2 jobs.

Reply:
This then?
```bash
make -j`nproc`
```

Reply:
`nproc` is part of GNU Core Utilities, native macOS analogue is  `sysctl -n hw.ncpu`, there is also a `hw.physicalcpu` and a `hw.logicalcpu` for hyper-threading.

Reply:
make -j `sysctl -n hw.ncpu`
make -j $((`sysctl -n hw.ncpu`+1))

Reply:
Done